src/Mmi/App/executables/php-serve-router.php

```diff
@@ -0,0 +1,28 @@
+<?php
```

Comment:
Wytłumaczę się z tego cudownego pliku, otóż:
"php -S" działa tak, że:
- jak jakiejś ścieżki w /web nie ma (np. /cmsAdmin) to wrzuca na index.php (so far so good)
- jak ścieżka phpowi przypomina jakiś plik np. /data/foto.jpg to próbuje go zaserwować nawet jeśli go nie ma i nie ma jak go zmusić żeby poszedł do index.php

Kluczem do działania wszystkich aplikacji na mmi jest to, że jak czegoś nie ma to ZAWSZE ma lecieć na apkę (index.php) i apka myśli co z tym zrobić

Nie znalazłem innego sposobu niż taki właśnie routerek.

Reply:
Dodam, że instaluję go w /web tylko w trybie "dev"